map_test.go

```diff
@@ -1091,38 +1103,50 @@ func TestMapIterate(t *testing.T) {
 	}
 	defer hash.Close()
 
-	if err := hash.Put("hello", uint32(21)); err != nil {
-		t.Fatal(err)
-	}
+	var (
+		key   string
+		value uint32
+		keys  []string
+		data  = []string{"hello", "world"}
+	)
 
-	if err := hash.Put("world", uint32(42)); err != nil {
-		t.Fatal(err)
+	for i, k := range data {
```

Comment:
Nit: for clarity's sake, keep variable declarations close to where they're used:
```suggestion
	data := []string{"hello", "world"}
	for i, k := range data {
```

(keep key, value, keys where they are)